lib/phase/permutation.js

```diff
@@ -22,16 +22,21 @@ module.exports = validatedDoc => (
                 annotations: Hoek.reach(doc, `jobs.${jobName}.annotations`, {
                     default: {}
                 }),
-                image: Hoek.reach(doc, `jobs.${jobName}.image`),
                 commands: Hoek.reach(doc, `jobs.${jobName}.commands`),
-                secrets: Hoek.reach(doc, `jobs.${jobName}.secrets`),
                 environment: Hoek.reach(doc, `jobs.${jobName}.environment`, {
                     default: {}
                 }),
+                image: Hoek.reach(doc, `jobs.${jobName}.image`),
+                secrets: Hoek.reach(doc, `jobs.${jobName}.secrets`),
                 settings: Hoek.reach(doc, `jobs.${jobName}.settings`, {
                     default: {}
                 })
             };
+
+            if ('description' in doc.jobs[jobName]) {
```

Comment:
Yeah I guess it doesn't make sense if the description is an empty string or a falsy value. I just kept the option in case the description field was provided but I'll change it.